.github/workflows/test.yml

```diff
@@ -41,6 +42,45 @@ jobs:
       run: pip install -r requirements-dev.txt
     - name: Run tests
       run: pytest
+  check_db:
```

Comment:
I think to make this optional it'd need to be a separate job rather than a step. (i.e. move into a separate workflow file)